apps/nextjs/src/components/post.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import type { inferRouterOutputs } from "@trpc/server";
+
+import type { AppRouter } from "@cfd/api";
```

Comment:
Use https://github.com/uoftblueprint/centre-for-dreams/blob/main/apps/nextjs/src/utils/api.ts#L76 for the router outputs